src/ublue_update/cli.py

```diff
@@ -160,8 +135,8 @@ def run_updates(system, system_update_available):
                     "DISPLAY=:0",
                     f"XDG_RUNTIME_DIR={xdg_runtime_dir}",
                     f"DBUS_SESSION_BUS_ADDRESS=unix:path={xdg_runtime_dir}/bus",
-                    "/usr/bin/ublue-update",
-                    "-f",
+                    "/usr/bin/topgrade",
+                    "--config /etc/ublue-update/topgrade.toml",
                 ],
                 capture_output=True,
             )
```

Comment:
This is running topgrade for every user logged onto the system, does topgrade manage multiuser? This also means topgrade won't be ran as root

Reply:
Topgrade by default is meant to be run by the current user and will elevate as needed,. Any suggestions on how we might best handle this?

Reply:
Opted for two configurations, now we can call topgrade multiple times. Once with the system config on the root user, and then as many times as necessary for each active user after.

Reply:
Also realizing I took out part of your script for the automatic rebasing, so I will restore that

Reply:
ok @gerblesh that's restored now, I opted to keep the /system folder and call topgrade from it to keep the very much needed rebase check